### ISSUE TITLE
Can you add new feature that we can drag a stone for study purpose?

### ISSUE THREAD

**Issue by jirapast** [SabakiHQ/Sabaki#613]

**Comment by yishn** [comment]
What do you mean by dragging a stone? Please elaborate.

**Comment by apetresc** [comment]
I'm almost certain he means moving already-placed stones around by click+dragging them from their current intersection to a new one, the way you would on a physical board.

(Not that I'm condoning this request)

**Comment by jirapast** [comment]
Exactly, draging already-placed stones by clicking like this software  "https://gowrite.net/"

**Comment by yishn** [comment]
This issue has been closed accidentally.
